src/utils/cookies.ts

```diff
@@ -67,6 +67,10 @@ function isSecureEnvironment(req: IncomingMessage): boolean {
     return false;
   }
 
+  if (process.env.AUTH0_ALLOW_INSECURE_COOKIE === 'true') {
```

Comment:
Would it be possible to make this code slightly more resilient? Eg:

```suggestion
  if (/^true$/i.test(process.env.AUTH0_ALLOW_INSECURE_COOKIE)) {
```